names/module.py

```diff
@@ -0,0 +1,210 @@
+
+    def __init__(self, bot: commands.Bot):
+        # Check if dependency module is loaded.
+        if not BaseAdminModule.get("boards.karma").enabled:
+            raise Exception(
+                "Unable to load names module due to missing module `boards.karma`."
+            )
+
+        self.bot = bot
+
+    @commands.bot_has_permissions(manage_nicknames=True)
+    @commands.guild_only()
+    @check.acl2(check.ACLevel.EVERYONE)
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @commands.group(name="nickname", aliases=["name"])
+    async def nickname_(self, ctx):
+        """Change your nickname"""
+        await utils.discord.send_help(ctx)
+
+    @commands.guild_only()
+    @check.acl2(check.ACLevel.EVERYONE)
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @nickname_.command(name="prices")
+    async def prices(self, ctx):
+        """Display prices for various services"""
+
+        embed = utils.discord.create_embed(
+            author=ctx.author,
+            title=_(ctx, "Change of nickname"),
+            description=_(
+                ctx,
+                "Values are in karma points. Use commnand **nickname/name** for more info.",
+            ),
+        )
+        price = Price.get(ctx.guild.id)
+        embed.add_field(
+            name=_(ctx, "Set"),
+            value=_(ctx, "Price **{price}**").format(
+                price=price.set_price if price is not None else "Not set"
+            ),
+        )
+        embed.add_field(
+            name=_(ctx, "Reset"),
+            value=_(ctx, "Price **{price}**").format(
+                price=price.reset_price if price is not None else "Not set"
+            ),
+        )
+
+        await ctx.reply(embed=embed)
+
+    @commands.guild_only()
+    @check.acl2(check.ACLevel.EVERYONE)
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @nickname_.command(name="set")
+    async def nickname_set(self, ctx, nickname: str):
+        """Change server nickname"""
+        if not BaseAdminModule.get("boards.karma").enabled:
+            await bot_log.error(
+                ctx.author,
+                ctx.channel,
+                "Repository boards.karma is not loaded. Cannot set nickname.",
+            )
+            await ctx.reply(
+                _(ctx, "Cannot change nickname because module karma is not loaded.")
+            )
+            return
+
+        if nickname == ctx.author.display_name:
+            await ctx.reply(
+                _(
+                    ctx,
+                    "New nickname {nickname} is same as current nickname. No need to change.",
+                ).format(nickname=nickname)
+            )
+            return
+
+        price = Price.get(ctx.guild.id)
+        if price is None:
+            await bot_log.debug(
+                ctx.author,
+                ctx.channel,
+                "Price for set command is not set.",
+            )
```

Comment:
We don't need to log this, the bot has not done anything, no state has been updated.

Reply:
Removed this and the same in second command